source/Calamari.AzureAppService/RetryPolicies.cs

```diff
@@ -18,5 +25,15 @@ public static class RetryPolicies
                                                                                                   .OrResult<HttpResponseMessage>(r => (int)r.StatusCode >= 500 || r.StatusCode == HttpStatusCode.RequestTimeout)
                                                                                                   .WaitAndRetryAsync(5,
                                                                                                                      retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)) + TimeSpan.FromMilliseconds(Jitterer.Next(0, 1000)));
+
+        public static RetryPolicy<HttpResponseMessage> AsynchronousZipDeploymentOperationPolicy { get; } = Policy.HandleResult<HttpResponseMessage>(r => r.StatusCode == HttpStatusCode.Accepted)
+                                                                                                                 .WaitAndRetryForeverAsync((_1,ctx) => TimeSpan.FromSeconds(2),
```

Comment:
Should we retry forever?

Reply:
In practice it doesn't. There just isn't a built-in `RetryAndWait` with a fixed expiration. As mentioned in the top, we wrap this in a `Timeout` policy which marks a cancellation token as cancelled, which then in turn aborts this policy.